### ISSUE TITLE
Documentation feedback: Links to pages are broken after refactoring

### ISSUE THREAD

**Issue by amotl** [crate/crate-docs-theme#492]
## Problem

For example at https://github.com/crate/cratedb-guide/issues/76, you can spot:

```
Page URL: https://cratedb.com/https://cratedb.com/docs/guide/integrate/etl/mongodb.html
```

This is wrong, because it includes the schema://host part twice.

## Thoughts

Most likely, this has been introduced by GH-488.

**Comment by amotl** [comment]
@msbt submitted a patch to fix the problem. Thanks!

- GH-493

**Comment by amotl** [comment]
Released with [version 0.32.2](https://github.com/crate/crate-docs-theme/releases/tag/0.32.2). Thanks again.
